docs/pages/setup/reference/config.mdx

```diff
@@ -89,6 +89,21 @@ teleport:
     # auth_token: /var/lib/teleport/tokenjoin
     auth_token: xxxx-token-xxxx
 
+    # join_params are parameters to set when joining a cluster via
+    # ec2, iam or a token.
+    #
+    # ref: https://goteleport.com/docs/setup/guides/joining-nodes-aws-ec2/
+    #      https://goteleport.com/docs/setup/guides/joining-nodes-aws-iam/
+    join_params:
+        # When join_method is "token", token_name is either the
+        # token or the path to a file containing the token.
+        #
+        # Otherwise token_name is the name of the predefined ec2 or
+        # iam token.
+        token_name:  "token-name"
```

Comment:
If the join method is `ec2` or `iam`, does the user need to supply a value here? If so, we should include these values in the comment instead of "the predefined ec2 or iam token".